conda_env/cli/main_create.py

```diff
@@ -51,22 +53,36 @@ def configure_parser(sub_parsers):
         '-q', '--quiet',
         default=False,
     )
+    p.add_argument(
+        'handle',
```

Comment:
ok, let's deprecate `--name` you are right about github.
I'm gonna focus on having:
```
conda env create --name malev/wakari
`--name` is being deprecated, just use:
    conda env create malev/wakari
```